compilers/concrete-optimizer/concrete-optimizer/src/dag/unparametrized.rs

```diff
@@ -204,7 +204,7 @@ impl<'dag> DagBuilder<'dag> {
         let op = Operator::LevelledOp {
             inputs,
             complexity,
-            manp,
+            weights: weights.as_ref().to_owned(),
```

Comment:
maybe an assert here that weights and inputs have the same length so it will not panic later